### DISCUSSION TITLE
SD next not support any more styles.csv

### DISCUSSION THREAD

**Discussion by ShmuelRonen** [vladmandic/sdnext#2162] [Q&A]
Hello...

I have a styles.csv file that I work a lot to make and arrange by categories for XDXL.
Starting with the latest versions, SD next to my surprise no longer supports this format but only the .json format and that too in separate files where each style is found separately.
Is there an option to the users to choose which styles format they want to use?

**Comment by vladmandic** [comment]
see #2148 and #2164

**Comment by ShmuelRonen** [comment]
Thanks!!!
I arranged each style separately in its own folder and now it's perfect.
![styles new](https://github.com/vladmandic/automatic/assets/80190186/3ea6c3f2-52c5-4e97-8427-d0787d88eadf)

**Reply by StrangerGithuber** [reply]
How did you do this? I create folder and add the style but mine doesn't look so organized as yours.

**Comment by larinius** [comment]
I set path to styles.csv - nothing happens, don't see any conversion happens. 
Why you made so much problems from nothing? 
So at the end you broke all good features of csv and added nothing in exchange.
![image](https://github.com/user-attachments/assets/d4d4fde5-6c1f-4192-ae08-c775146f108d)

**Reply by vladmandic** [reply]
commenting on an issue closed a year ago does not help. if you have a problem, document it and create a ticket.

> Why you made so much problems from nothing?

because i want to move forward and not be stuck in the past. if you want old behavior, feel free to use old version.

>So at the end you broke all good features of csv and added nothing in exchange.

new styles engine has so much more features and functionality.
